src/lib/radio/radio.ts

```diff
@@ -516,4 +498,20 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  /** Function is called whenever the focus changes for the input element. */
+  private _onInputFocusChange(focusOrigin: FocusOrigin) {
```

Comment:
I'm not sure that calling it a "focus change" is accurate, `onInputFocus` should be fine.